ruby-specs/action.yml

```
@@ -0,0 +1,115 @@
+name: "Scalingo Ruby Specs GitHub Action"
+description: "Execute Ruby specs with code coverage in a GitHub Action"
+
+inputs:
+  mongodb:
+    description: "Start MongoDB if needed. Possible values: true | false | auto"
+    required: false
+    default: "auto"
+  redis:
+    description: "Start Redis if needed. Possible values: true | false | auto"
+    required: false
+    default: "auto"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4
```

issue: last version is v6. Is there a reason why you don't use it?
```suggestion
    - uses: actions/checkout@v6
```